tests/release/conftest.py

```diff
@@ -0,0 +1,158 @@
+import functools
+import os
+import time
+from collections.abc import Callable
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+import pytest
+from memory_profiler import memory_usage  # type: ignore
```

Comment:
mypy fails because there are no stubs or type hints for memory_profiler

Reply:
Add the specific ignore type

Reply:
It looks this is as specific as we can get, according to the 1st item in the 2nd numbered list in [this documentation](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker)

Reply:
Interesting, and so you're saying that you didn't get a `[XXX]` at the end of the mypy error message for this, e.g. 

`error: Argument 3 to "initializes_simulants" of "PopulationInterface" has incompatible type "**dict[str, list[str | Resource]]"; expected "str | Sequence[str]"  [arg-type]`